src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1320,7 +1321,7 @@ static uct_iface_ops_t uct_dc_mlx5_iface_tl_ops = {
     .iface_close              = UCS_CLASS_DELETE_FUNC_NAME(uct_dc_mlx5_iface_t),
     .iface_query              = uct_dc_mlx5_iface_query,
     .iface_get_device_address = uct_ib_iface_get_device_address,
-    .iface_is_reachable       = uct_dc_mlx5_iface_is_reachable,
+    .iface_is_reachable       = uct_ib_iface_is_reachable,
```

Comment:
why? we still have to check dc address type as it was before

Reply:
to deduplicate, now `uct_ib_iface_is_reachable()` directly calls `uct_ib_iface_is_reachable_v2()`, that is now up-to-date.

Reply:
actually, it should be `uct_iface_is_reachable_v2()` instead of the IB specific.

Reply:
maybe create a common wrapper, like `uct_base_ep_connect_to_ep` then?

Reply:
moved to base as `uct_base_iface_is_reachable()`